CHANGELOG.md

```diff
@@ -8,6 +8,9 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ### Added
 ### Fixed
+- Sampler.n_effective is no longer unnecessarily computed when sampling with
+  an infinite limit on n_effective.
+- Setting n_effective for .run_nested() and .sample_initial() is deprecated.
```

Comment:
n_effective is only deprecated for the static nested sampler!

Reply:
I've adjusted this (and added a "changed" section since deprecating a feature is not exactly fixing).